public/data/anevski-stefan.json

```
@@ -0,0 +1,12 @@
+{
+ "name": "Stefan Anevski",
+ "location": "Skopje, North Macedonia",
+ "bio": "I'm a final year computer science and engineering student with a very big passion towards software engineering.",
+ "avatar": "https://github.com/anevski-stefan.png",
+ "portfolio": "https://stefananevski.xyz/",
+ "skills": ["Java", "Javascript", "Django", "Typescript", "Storybook", "HTML", "CSS", "TaiwindCSS", "Bootstrap", "React", "Handlebars", "NodeJS", "Express", "PostgreSQL", "MySQL", "PHP"],
+ "social": {
+   "GitHub": "https://github.com/anevski-stefan",
+   "LinkedIn": "https://www.linkedin.com/in/stefan-anevski/"
+ }
+},
```

```suggestion
}
```
Remove the last comma from your profile file.